tools/proto-js-plugin/src/main/java/io/spine/js/generate/Statements.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.js.generate;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static java.lang.String.format;
+
+/**
+ * The utilities for creation of typical statements.
+ */
+public class Statements {
+
+    /** Prevents instantiation of this utility class. */
+    private Statements() {
+    }
+
+    /**
+     * Composes a statement returning a string literal.
+     */
+    public static String returnString(String literalValue) {
+        checkNotNull(literalValue);
+        String literal = format("'%s'", literalValue);
+        String statement = returnValue(literal);
+        return statement;
+    }
+
+    /**
+     * Composes a statement returning the value.
+     */
+    public static String returnValue(Object value) {
+        checkNotNull(value);
+        String statement = "return " + value + ';';
+        return statement;
+    }
+
+    /**
+     * Composes a map entry with the string literal key.
+     */
+    public static String mapEntry(String key, Object value) {
+        return "['" + key + "', " + value + ']';
```

Comment:
Shouldn't there be a `:` symbol between the key and the value?

Reply:
It's fine, the same in the [documentation](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Map#%D0%92%D0%B7%D0%B0%D0%B8%D0%BC%D0%BE%D0%BE%D1%82%D0%BD%D0%BE%D1%88%D0%B5%D0%BD%D0%B8%D1%8F_%D1%81_%D0%BE%D0%B1%D1%8A%D0%B5%D0%BA%D1%82%D0%BE%D0%BC_Array).